src/dom-to-image-more.js

```diff
@@ -238,9 +291,22 @@
             });
 
         function newCanvas(node, scale) {
+            let width = options.width || util.width(node);
+            let height = options.height || util.height(node);
+
+            // per https://www.w3.org/TR/CSS2/visudet.html#inline-replaced-width the default width should be 300px if height
+            // not set, otherwise should be 2:1 aspect ratio for whatever height is specified
+            if (util.isDimensionMissing(width)) {
+                width = util.isDimensionMissing(height) ? 300 : height * 2.0;
```

Comment:
According to the spec, if there's no `width`, but there is a `height`, set the `width = 2 * height`. If there's no `height` *either*, then fallback to the [standards-defined](https://www.w3.org/TR/CSS2/visudet.html#inline-replaced-width) intrinsic `width` of 300px.

Next, we handle missing `height` by just calculating it at the [standards-defined](https://www.w3.org/TR/CSS2/visudet.html#inline-replaced-width) `2:1 aspect ratio` (so divide `width` by 2).